lib/modules/platform/pr-body.ts

```diff
@@ -21,21 +25,35 @@ export function hashBody(body: string | undefined): string {
   return result;
 }
 
-export function isRebaseRequested(body: string | undefined): boolean {
+function isRebaseRequested(body: string | undefined): boolean {
   return !!body?.includes(`- [x] <!-- rebase-check -->`);
 }
 
+export function getRenovatePrVerData(body: string): string | undefined {
+  const match = prVerDataRe.exec(body);
+  return match?.groups?.payload;
+}
+
 export function getPrBodyStruct(
   input: string | undefined | null
 ): PrBodyStruct {
-  const str = input ?? '';
-  const hash = hashBody(str);
+  const body = input ?? '';
+  const hash = hashBody(body);
   const result: PrBodyStruct = { hash };
 
-  const rebaseRequested = isRebaseRequested(str);
+  const rebaseRequested = isRebaseRequested(body);
   if (rebaseRequested) {
     result.rebaseRequested = rebaseRequested;
   }
 
+  // istanbul ignore if
+  if (!is.string(body)) {
+    return result;
+  }
```

Comment:
remove, body is always defined

Reply:
it falls on null checks  :/ when i call the method below using body, i can let the function accept undefined and return undefined and that might solve it :/

Reply:
that's strange because `body = input ?? ''`, so it should always `string` 🤔